src/ZombieDriver.php

```diff
@@ -236,7 +236,16 @@ public function setRequestHeader($name, $value)
      */
     public function getResponseHeaders()
     {
-        return (array) $this->server->evalJS('browser.window._response.headers', 'json');
+        $js = <<<JS
+        if (true == zombieVersionCompare('4.0.0', '>=')) {
+          stream.end(JSON.stringify(browser.window._response.headers.toObject()));
```

Comment:
``browser.response`` is just calling ``browser.window._response`` internally: https://github.com/assaf/zombie/blob/0382cf3442a62db4dea0cca563335137e3ad7083/src/index.js#L1154.
But it is a public API, which is better than accessing the internal ``_response`` property (and no, ``browser.response`` does not exist in 2.x or 3.x)